Components/cards.css

```diff
@@ -0,0 +1,133 @@
+.card-vertical {
+  max-width: 230px;
+  padding: 0.5rem 1rem 0 1rem;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 1px 4px 5px 3px var(--light-secondary-color);
+  border-radius: 0.4rem;
+}
+.card-img {
+  display: flex;
+  max-height: 210px;
+}
+.card-badge {
+  background-color: var(--primary-color);
+  color: var(--complementary-text-color);
+  padding: 0 0.3rem;
+  max-width: 4.5rem;
+  border-radius: 0.2rem;
+  position: absolute;
+  top: 1rem;
+  left: 0.2rem;
+  border: 1px solid var(--complementary-text-color);
+}
+.card-favourite-icon {
+  background-color: #fff;
+  padding: 0.4rem;
+  border-radius: 50%;
+  position: absolute;
+  right: 1rem;
+  top: 0.5rem;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.card-details {
+  display: flex;
+  flex-direction: column;
+  margin: 0.5rem 0.3rem;
+}
+.card-title,
+.card-category,
+.card-description,
+.card-price {
+  margin-top: 0.25rem;
+}
+.card-title {
+  color: var(--dark-primary-color);
+}
+
+.card-category {
+  color: var(--dark-secondary-color);
+}
+
+.card-price {
+  color: var(--light-primary-color);
+}
+
+.card-vertical .btn {
+  margin: 0.5rem 0 0 0;
+}
+
+.overlay-container {
+  opacity: 0.9;
+}
+
+.overlay-text {
+  position: absolute;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(234, 234, 236, 0.6);
+}
+
+.overlay-text span {
+  background-color: var(--secondary-color);
+  padding: 1rem;
+}
+
+.card-horizontal {
+  max-width: 800px;
+  padding: 0.5rem 1.8rem 0 1.5rem;
+  display: flex;
+  box-shadow: 1px 4px 5px 3px var(--light-secondary-color);
+  position: relative;
+}
+.card-horizontal .card-img {
+  width: 120px;
```

Comment:
Don't use absolute units if you plan to make them responsive.

Reply:
Changes made